app/konnect/runtime-manager/kong-gateway-runtime.md

```diff
@@ -59,11 +58,186 @@ representative for access.
+      kong reload exit" | docker exec -i <kong-container-id> /bin/sh
+    ```
+
+    Or, if bringing up a new container:
+
+    ```sh
+    $ docker run -d --name kong-gateway-dp1 --network=kong-gateway-net \
+      -e "KONG_ROLE=data_plane" \
+      -e "KONG_DATABASE=off" \
+      -e "KONG_ANONYMOUS_REPORTS=off" \
+      -e "KONG_CLUSTER_MTLS=pki" \
+      -e "KONG_CLUSTER_CONTROL_PLANE=<example.cp.konnect.foo>:443" \
+      -e "KONG_CLUSTER_SERVER_NAME=<kong-cpoutlet-example.service>" \
+      -e "KONG_CLUSTER_TELEMETRY_ENDPOINT=<example.tp.konnect.foo>:443" \
+      -e "KONG_CLUSTER_TELEMETRY_SERVER_NAME=<kong-telemetry-example.service>" \
+      -e "KONG_CLUSTER_CERT=/<path-to-file>/cluster.crt" \
+      -e "KONG_CLUSTER_CERT_KEY=/<path-to-file>/cluster.key" \
+      -e "KONG_LUA_SSL_TRUSTED_CERTIFICATE=/<path-to-file>/ca.crt" \
+      --mount type=bind,source="$(pwd)"/cluster,target=<path-to-keys-and-certs>,readonly \
+      -p 8000:8000 \
+      -p 8001:8001 \
+      kong-gateway-dp1
+    ```
+
+5. On the **Configure New Runtime** page, click **Done** to go to the Runtime
+Manager overview.
+
+    The Runtime Manager will include a new entry for your instance.
+
+{% endnavtab %}
+{% navtab Using kong.conf %}
+
+1. (Optional, if no gateway data plane exists) Find the documentation for
+[your platform](/enterprise/latest/deployment/installation),
+and follow the instructions in Steps 1 and 2 **only** to download and install
+{{site.base_gateway}} 2.3.0.0.
+
+    <div class="alert alert-warning">
+        Do not start or create a database on this node.
+    </div>
+
+2. In the **Configuration Parameters** section, copy the codeblock.
+
+3. Replace the values in `KONG_CLUSTER_CERT`, `KONG_CLUSTER_CERT_KEY`,
+and `KONG_CLUSTER_CA_CERT` with the paths to your certificate files.
+
+4. Open your instance's `kong.conf` file. Remove the `KONG_` prefix from the
+parameters in the sample codeblock and add the parameters to the file:
+
+    ```sh
+    role = data_plane
+    database = off
+    anonymous_reports = off
+    cluster_mtls = pki
+    cluster_control_plane = <example.cp.konnect.foo>:443
+    cluster_server_name = <kong-cpoutlet-example.service>
+    cluster_telemetry_endpoint = <example.tp.konnect.foo>:443
+    cluster_telemetry_server_name = <kong-telemetry-example.service>
+    cluster_cert = /<path-to-file>/cluster.crt
+    cluster_cert_key = /<path-to-file>/cluster.crt
+    lua_ssl_trusted_certificate = /<path-to-file>/ca.crt
+    ```
+
+    See [Parameters](#parameters) for descriptions and the matching fields
+    in {{site.konnect_short_name}}.
+
+5. Restart {{site.base_gateway}} for the settings to take effect:
+
+    ```sh
+    $ kong restart
+    ```
+
+6. On the **Configure New Runtime** page, click **Done** to go to the Runtime
+Manager overview.
+
+    The Runtime Manager will include a new entry for your instance.
+
+{% endnavtab %}
+{% endnavtabs %}
+
+{% endnavtab %}
```

Comment:
Should there be two sets of these ?

Reply:
yep - it's closing two different navtabs. Usually I put a comment in here to say something like `<!-- end of codeblock navtabs -->`, and I didn't this time, so I'll add one in :)